bin/latest-stable

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# list all elixir releases
+# reject releases before 1.0.0 (starting with 0)
+# reject main and master branches
+# reject releases with otp version
+# reject release candidates
+# finally take last one
+ASDF_ELIXIR_LATEST_VERSION=$(asdf list all elixir | grep -Pv "^0|^main|^master|otp|rc|^$" | tail -n 1)
```

Comment:
Rather than invoking this callback via asdf it'd probably be better to just invoke this plugin's `list-all` script directly: `./list-all | grep -Pv "^0|^main|^master|otp|rc|^$" | tail -n 1`

Reply:
@Stratus3D makes sense, used your suggestion and added detailed the comments